src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Large.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Large
```

Comment:
It's not clear to me why this class/benchmarks is needed.
It seems cleaner to reuse the Perf.Basic above with two different "DataSize" (small/large).

Reply:
This can be merged once we remove the pool option. I split it since I believe I was hitting the 16 benchmark limit.

Reply:
I would prefer not to add this one, and add later the good benchmark.